lib/jsdom/browser/xhr/utils.js

```diff
@@ -0,0 +1,235 @@
+"use strict";
+
+var request = require("request");
+var url = require("url");
```

Comment:
The built-in URL module is not allowed in jsdom. Instead use the URL constructor from util.js